backend/commons/src/main/java/com/micifuz/commons/configuration/ConfigManager.java

```diff
@@ -17,11 +17,13 @@
 
     private static final String ENV_SEPARATOR = "_";
     private static final String ENV_PREFIX = "X";
-    private static final String DEFAULT_CONFIG_PATH = "config.yaml";
+    private static final String DEFAULT_CONFIG_PATH = "simpleTest-config.yaml";
```

Comment:
Probably this is not what you aimed for, it's estrange to have a default config value as simpleTest-config.yaml

Reply:
Probably a refactor thing